client/webserver/site/src/html/dexsettings.tmpl

```diff
@@ -2,6 +2,7 @@
 {{template "top" .}}
 {{$passwordIsCached := .UserInfo.PasswordIsCached}}
 <div id="main" data-handler="dexsettings" data-host="{{.Exchange.Host}}" class="text-center py-5 overflow-y-auto">
+  <div class="go-back-to-settings"><span id="goBackToSettings" class="ico-wide-headed-left-arrow fs24"></span></div>
   <span class="settings-gear ico-settings"></span>
```

Comment:
Please consider these changes and then deleting the `go-back-to-settings` class.
```suggestion
<div id="main" data-handler="dexsettings" data-host="{{.Exchange.Host}}" class="flex-center flex-column align-items-stretch py-5 mx-auto">
  <div id="goBackToSettings" class="d-flex justify-content-start align-items-center pointer hoverbg py-2">
    <span class="ico-wide-headed-left-arrow fs24 lh1"></span>
  </div>
  <div class="flex-center pb-2"><span class="settings-gear ico-settings"></span></div>
```

Reply:
Thanks for the suggestion. I've updated this file with minor modification to your suggestion.